client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -768,6 +770,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                         </TableRow>
                                         {checkGroupedInvestigationOpen.includes(indexedRow.epidemiologyNumber) &&
                                             allGroupedInvestigations.get(indexedRow.groupId)?.filter((row: InvestigationTableRow) => row.epidemiologyNumber !== indexedRow.epidemiologyNumber).map((row: InvestigationTableRow, index: number) => {
+                                                const currentGroupedInvestigationsLength = allGroupedInvestigations.get(indexedRow.groupId)?.filter((row: InvestigationTableRow) => row.epidemiologyNumber !== indexedRow.epidemiologyNumber).length!;
```

Comment:
if i got it right, you can just do length -1

Reply:
yea , it's more readable like that , added [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/962/commits/87d7b84fac39bc8a1de34b345cdc66101ded4ba5)

Reply:
changed to len [here](489f80d)